tests/test_logging.py

```diff
@@ -14,6 +19,19 @@ def has_handler(logger, handler_name):
     return any([handler.name == handler_name for handler in logger.handlers])
 
 
+def capture_stdout(command, *args, **kwargs):
+    normal_stdout = sys.stdout
+    captured_stdout = StringIO.StringIO()
+    sys.stdout = captured_stdout
+
+    command(*args, **kwargs)
+
+    captured_stdout.seek(0)
+    return captured_stdout.read()
+
+    sys.stdout = normal_stdout
```

Comment:
You probably mean to do this before the `return` (or actually in a `finally` block)